__tests__/__snapshots__/check-fixtures.js.snap

```diff
@@ -147,11 +147,70 @@ exports[`linting javascript.js fixture matches snapshot 1`] = `
     "ruleId": "no-undef",
     "severity": 2,
   },
+  {
+    "column": 121,
+    "endColumn": 121,
+    "endLine": 45,
+    "fix": {
+      "range": [
+        894,
+        894,
+      ],
+      "text": "
+",
+    },
+    "line": 45,
+    "message": "Insert \`⏎\`",
+    "messageId": "insert",
+    "nodeType": null,
+    "ruleId": "prettier/prettier",
+    "severity": 2,
+  },
 ]
 `;
 
 exports[`linting typescript.ts fixture matches snapshot 1`] = `
 [
+  {
+    "column": 48,
+    "endColumn": 61,
+    "endLine": 1,
+    "line": 1,
+    "message": "'three' is defined but never used.",
+    "messageId": "unusedVar",
+    "nodeType": "Identifier",
+    "ruleId": "@typescript-eslint/no-unused-vars",
+    "severity": 2,
+  },
```

Comment:
It looks like the `@typescript-eslint/no-unused-vars` rule now catches unused parameters as well.

```ts
export function add( one: number, two: number, three: String ): number {
	return one + two;
}
```